soepy/python/pre_processing/model_processing.py

```diff
@@ -38,7 +38,23 @@ def expand_init_dict(init_dict):
     ]
     shocks_cov = [shocks_cov[0] ** 2, shocks_cov[1] ** 2, shocks_cov[2] ** 2]
 
-    init_dict["DERIVED_ATTR"] = {"educ_range": educ_range, "shocks_cov": shocks_cov}
+    # Extract the number of types
+    type_shares_non_baseline = [
```

Comment:
Appears `v`is not used. If so, please replace with `_`.

Reply:
Done